History.md

```diff
@@ -1,5 +1,9 @@
 ## 0.8.0 / 2017-11-20
 
+* Breaking Change
+  * In `UploadedFile.new`, when passing a `Pathname`, an additional argument
+    is now required (`original_filename`). Provide the new argument or
+    call `to_s` on your `Pathname`.
```

Comment:
What is the error you get if you don't do this? Is it the `nil` error we are seeing in #207?

Reply:
The error message is "Missing original_filename for IO". It comes from `initialize_from_io`.

```
      def initialize_from_io(io, original_filename)
        @tempfile = io
        @original_filename = original_filename || raise(ArgumentError, 'Missing `original_filename` for IO')
      end
```

Reply:
OK, interesting. There seems to be a couple of issues there, because people have also reported of  `nil` error coming:

```
NoMethodError:
       undefined method `size' for nil:NilClass
```

Could you update the text in line with this, so that this error is also listed there? Thanks a lot!